eemeter/visualization.py

```diff
@@ -49,14 +49,19 @@ def plot_time_series(meter_data, temperature_data, **kwargs):
         Tuple of ``(ax_meter_data, ax_temperature_data)``.
     """
     # TODO(philngo): include image in docs.
-    try:
-        import matplotlib.pyplot as plt
-    except ImportError:  # pragma: no cover
-        raise ImportError("matplotlib is required for plotting.")
-
-    default_kwargs = {"figsize": (16, 4)}
-    default_kwargs.update(kwargs)
-    fig, ax1 = plt.subplots(**default_kwargs)
+    figure = kwargs.pop('figure')
```

Comment:
This seems to be breaking some tests - perhaps adding a default (e.g., `None`) would fix?